BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -179,69 +179,112 @@ extension StackLayout {
 
 }
 
+// MARK: - Layout logic
+
+// Stack layout is generalized to be independent of the stack's axis (which is `horizontal` for rows and `vertical`
+// for columns).
+//
+// This terminology is used to symbolically represent the two axes:
+//
+//   - The axis along which elements are being laid out is generally called "axis".
```

Comment:
Minor, but it would be good to include examples of this in practice (eg, what is axis and cross in a row vs. column). I still have to think about it every damn time...

Reply:
I just added some suggestions with ascii art, maybe those should be moved/copied up to here as well.

Reply:
Will do.
- [ ] add ascii art docs for axis/cross